src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -176,6 +177,7 @@ public void windFlywheel(double rpm) {
     if (rpm == 0) {
       flywheelPID.setReference(0, CANSparkMax.ControlType.kVoltage);
       flywheelPID.setIAccum(0);
+      Shuffleboard.addEventMarker("flyWheel Starts", EventImportance.kHigh);
```

Comment:
This line is if the RPM is set to 0.

Would move it before the if.  Maybe it would be good to also capture the RPM?

Reply:
Changed accordingly, and RPM has been captured on SmartDashboard